script.js

```diff
@@ -1,5 +1,91 @@
+
+// converts to am and pm
+function timeConversion(timeString){
+    const usTime = timeString.substring(0,2);
+    const minutes = timeString.substring(3,5);
+    let numTime = Number(usTime);
+    let period = "";
+    if(numTime === "00:"){
+        numTime += 12;
+        period += "AM";
+    }
+    else if(numTime < 12){
+        period += "AM";
+    }
+    else{
+        numTime -= 12;
+        period += "PM";
+    }
+    return `${numTime}:${minutes} ${period}`
+}
```

Comment:
Very American of you to convert standard international 24h time to 12hr time (haha, just kidding... nice job with this function!)